examples/recipes/role-based-access/schema/Orders.js

```diff
@@ -0,0 +1,28 @@
+cube(`Orders`, {
+  sql: `SELECT * FROM public.orders`,
+  
+  measures: {
+    count: {
+      type: `count`,
+      drillMembers: [id, createdAt]
+    }
+  },
+  
+  dimensions: {
+    id: {
```

Comment:
@rchkv Do we need `id` and `createdAd` as well as `drillMembers` on `count` for this example? If not, I suggest removing them.